packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/CreateFolderControls.tsx

```diff
@@ -6,7 +6,14 @@ import { useControl } from '../../context/controls';
 import { Controls } from '../Controls';
 import { Title } from './Controls';
 
-const { Exit, Primary, Target } = Controls;
+const { Exit, Message, Primary, Target } = Controls;
+
+export const isValidFolderName = (name: string | undefined): boolean => {
+  return name !== undefined && name.length > 1 && name.endsWith('/');
```

Comment:
Think this can be simplified to:
```suggestion
  return !!name?.length && name.endsWith('/');
```

Reply:
I added the length > 1 incase we don't want folks to be able to upload folders named '/' (not sure if that's valid? It definitely still worked when I tried it). But I can take that out if so and just check for it being defined and ending with '/'

Reply:
An empty string evaluates to `false`:
<img width="156" alt="CleanShot 2024-08-13 at 13 32 28@2x" src="https://github.com/user-attachments/assets/070b5f0a-e50f-4079-aee4-9f735c46dfc4">

Reply:
Right, I think my question is more like if '/' is a valid name; it has length, and has a trailing slash, and is not undefined, but then the folder created would just be named '/' like in this case (where I added a folder with just '/' inside of the existing Cloudberry folder). 

<img width="215" alt="Screenshot 2024-08-13 at 4 38 39 PM" src="https://github.com/user-attachments/assets/1fc9de70-8f85-474f-ae94-11cfa74811a5">

Adding a check that the length is > 1 means you wouldn't be able to do that. Do we need to check for that?

Reply:
Oh i see, yeah that makes sense. Think we can still simplify it to:
```
const isValidFolderName = (name: string | undefined): boolean => !!name?.length > 1 && name.endsWith('/');
```
But good either way, just less code paths to test (maybe)